livekit-rtc/livekit/rtc/audio_mixer.py

```diff
@@ -184,6 +184,10 @@ async def _get_contribution(
             except StopAsyncIteration:
                 exhausted = True
                 break
+            # AudioStream may yield either AudioFrame or AudioFrameEvent; unwrap if needed
```

Comment:
The correct typing should be:

`self, stream: AsyncIterator[AudioFrame | AudioFrameEvent], buf: np.ndarray`